apps/cli/src/live-common-setup.ts

```diff
@@ -210,5 +209,4 @@ if (!process.env.CI) {
 export function closeAllDevices() {
   Object.keys(cacheBle).forEach(disconnect);
   closeAllSpeculosDevices();
-  disconnectAll();
```

Comment:
You should call the `coin-polkadot` disconnect method.
You should not remove code behavior without a test.